schema/ThematicBreak.schema.yaml

```diff
@@ -3,6 +3,7 @@ title: ThematicBreak
 extends: Entity
 role: secondary
 status: stable
+category: metadata
```

Comment:
prose

Reply:
Thanks for clarifying that for me, I have made those changes.